src/playground/views/main.handlebars

```diff
@@ -0,0 +1,215 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<link rel="apple-touch-icon" sizes="180x180" href="/img/apple-touch-icon.png">
+	<link rel="icon" type="image/png" sizes="32x32" href="/img/favicon-32x32.png">
+	<link rel="icon" type="image/png" sizes="16x16" href="/img/favicon-16x16.png">
+
+	{{#if gtm}}
+	<!-- Global site tag (gtag.js) - Google Analytics -->
+	<script async src="https://www.googletagmanager.com/gtag/js?id={{gtm}}"></script>
+	<script>
+		window.dataLayer = window.dataLayer || [];
+		function gtag() { dataLayer.push(arguments); }
+		gtag('js', new Date());
+		gtag('config', '{{gtm}}');
+	</script>
+	{{/if}}
```

Comment:
Not sure if adding tracking code in a public repository is okay... This would mean that we need to get users consent for storing their tracking data

Reply:
@Tuhaj , what do you think?

Reply:
Moved to a separate https://github.com/xFAANG/askql/issues/235